spark/src/main/scala/org/apache/mahout/drivers/MahoutSparkDriver.scala

```diff
@@ -76,6 +76,10 @@ abstract class MahoutSparkDriver extends MahoutDriver {
     if (!_useExistingContext) {
       sparkConf.set("spark.kryo.referenceTracking", "false")
         .set("spark.kryoserializer.buffer.mb", "200")// this is default for Mahout optimizer, change it with -D option
+        // the previous has been marked deprecated as of Spark 1.4 by the below line,
+        // remove the above line when Spark finally retires above for below
+        .set("spark.kryoserializer.buffer", "200")
+
 
```

Comment:
I think you have to put the units in the string
   
    .set("spark.kryoserializer.buffer", "200m")

would be the equivalen iirc